test/datahike/test/core_test.cljc

```diff
@@ -116,60 +116,3 @@
           (dc/db-with (dc/empty-db) [{:a 1 :b 2 :c 4} {:a 1}])
           (dc/db-with (dc/empty-db) [{:a 1 :b 3 :d 5}])))))
 
-;; connection tests
```

Comment:
Why are these tests gone now?

Reply:
Because you cannot instantiate connections directly anymore, you need to go through the connector now (which deduplicates and ensures that there is always only one connection to a store.

Reply:
These tests were also not really providing any important coverage as far as I could see.

Reply:
I see, I believe you are right about the coverage.